src/v/config/configuration.cc

```diff
@@ -1836,22 +1836,35 @@ configuration::configuration()
   , cloud_storage_max_partition_readers_per_shard(
       *this,
       "cloud_storage_max_partition_readers_per_shard",
-      "Maximum concurrent partition readers of remote data per CPU core.  If "
-      "unset, "
-      "value of `topic_partitions_per_shard` is used, i.e. one reader per "
-      "partition if the shard is at its maximum partition capacity.  These "
-      "readers have the"
-      "lifetime of a Kafka consume request, so this property controls how many "
-      "consume"
-      "requests to remote data can make progress at the same time.",
+      "Maximum partition readers per shard (deprecated)",
```

Comment:
Should `cloud_storage_max_segment_readers_per_shard` and `cloud_storage_max_materialized_segments_per_shard` be deprecated too?

Reply:
The first one is not deprecated (it can be used to change the memory limit to the lower side, you can set it to something and if that many readers require less memory than the memory limit will be lowered). This is a safety hatch in case if we're still running into memory issues or in case if RP has huge amount of memory per shard and we don't need as much.
The second one is deprecated.

Reply:
I missed the second one. Oke, let's keep `cloud_storage_max_segment_readers_per_shard`.